config/routes.rb

```diff
@@ -29,6 +28,14 @@
   get "/profile", to: 'profile#index'
 
   get "/login", to: 'login#index'
+  get "/login", to: 'sessions#new'
+  post "/login", to: 'sessions#create'
 
```

Comment:
This functionality will be super helpful to test app in the browser with seeded users!